src/bindings/python/src/openvino/utils/node_factory.py

```diff
@@ -117,12 +124,18 @@ def _(self, extension: Union[Extension, List[Extension]]) -> None:
 
     @staticmethod
     def _arguments_as_outputs(arguments: List[Union[Node, Output]]) -> List[Output]:
-        outputs = []
+        """Convert a list of Nodes and Outputs to a list of only Outputs.
+
+        :param      arguments:  The arguments to convert.
+        :return:    A list containing only Output objects.
+        """
+        outputs: List[Output] = []
         for argument in arguments:
-            if issubclass(type(argument), Output):
-                outputs.append(argument)
+            if isinstance(argument, Output):
+                outputs.append(cast(Output, argument))
             else:
-                outputs.extend(argument.outputs())
+                node_outputs = cast(Node, argument).outputs()
+                outputs.extend(node_outputs)
```

Comment:
Could you please discard changes here?
According to the list of errors these code should be fine